src/app/login-signup/register/register.component.ts

```diff
@@ -1,10 +1,62 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators , FormControl} from '@angular/forms';
+import { ReactiveFormsModule } from '@angular/forms';
+
+// an interface that contains the properties of username ,name, confirm password and password.
+interface User {
+  username: string,
+  name:string,
```

Comment:
Make consistency in spacing. check this to all
name:string -> name: string

Reply:
Changes are done